spec/factories/users.rb

```diff
@@ -1,4 +1,7 @@
 FactoryBot.define do
   factory :user do
+    sequence(:email) { |n| "person#{n}@example.com" }
+    sequence(:password) { |n| "amazing_secure_password#{n}" }
+    sequence :id
```

Comment:
The id will be assigned by the database (as auto increment) and does not need to be a FactoryBot sequence. This could make things more confusing, as a non-persisted `User` may suddenly have an id. This could also potentially confuse some of the inner magic of `ActiveRecord`.

Reply:
Roger. 

Fixed.